cpp/box/Box.h

```diff
@@ -380,6 +380,75 @@ class Box
         });
     }
 
+    //! Calculate distance between two points using boundary conditions
+    /*! \param r_i Position of first point
+        \param r_j Position of second point
+    */
+    float computeDistance(const vec3<float>& r_j, const vec3<float>& r_i) const
```

Comment:
Make this match the order of the docstring and also "from r_i to r_j" convention.
```suggestion
    float computeDistance(const vec3<float>& r_i, const vec3<float>& r_j) const
```